reconcile/utils/aws_api.py

```diff
@@ -67,7 +67,8 @@ def init_users(self):
         self.users = {}
         for account, s in self.sessions.items():
             iam = s.client('iam')
-            users = [u['UserName'] for u in iam.list_users()['Users']]
+            users = self.paginate(iam, 'list_users', 'Users')
+            users = [u['UserName'] for u in users]
```

Comment:
nit: why redefining the `users` variable instead of directly doing
```
users = [u['UserName'] for u in self.paginate(iam, 'list_users', 'Users')]
```

Reply:
I thought redefining users (or another var) is clearer whereas executing the function inside the list comprehension is a bit harder on the eye. also line length (can be split of course)

i'll leave it as it is for the sake of clarity and readability